apps/aws-amplify/src/locations/Sidebar.tsx

```diff
@@ -50,9 +52,10 @@ const Sidebar = () => {
         isFullWidth={true}
         isLoading={isLoading}
         isDisabled={isLoading}
-        onClick={buildAmplifyApp}>
-        {isLoading ? 'Building' : 'Build website'}
+        onClick={handleBuildAppActionCall}>
+        {isLoading ? 'Initiating build...' : 'Build now'}
```

Comment:
Eventually, we should add some error handling here and useful messages to the user. I am getting a `404 AppAction does not exist` and the button is stuck in the `Initiating build` state.

Reply:
This is tricky with the hosted app actions currently there is no documentation and they dont execute locally 🤔 this will definitely need to be handled at some point beyond the limited MVP we have flushed out here